lib/assets/connection_cell/main.js

```diff
@@ -257,16 +281,28 @@ export function init(ctx, info) {
         :grow
         :required
       />
+      <BaseInput
+        name="workgroup"
+        label="Workgroup"
+        type="text"
+        v-model="fields.workgroup"
+        inputClass="input"
+        :grow
+        :required="!!areFieldsEmpty(fields.workgroup, fields.output_location)"
+      />
       <BaseInput
         name="output_location"
         label="Output Location"
         type="url"
         v-model="fields.output_location"
         inputClass="input"
         :grow
-        :required
+        :required="!!areFieldsEmpty(fields.output_location, fields.workgroup)"
       />
     </div>
+    <small class="help-box">
+      You must use your AWS Credentials or authenticate your machine with <strong>aws</strong> CLI authentication.
```

Comment:
Should we mention about using `aws` cli is only if you add `aws_credentials` as dependency?

Also, should we ship `aws_credentials` to hex before adding that? Because users could add the package through the Livebook UI

Reply:
Yeah, sounds good!